atomix/cluster/src/main/java/io/atomix/raft/roles/PassiveRole.java

```diff
@@ -348,7 +347,16 @@ private void addSnapshotListener() {
                 .build()));
   }
 
+  private void setNextExpected(final ByteBuffer nextChunkId) {
+    nextPendingSnapshotChunkId = nextChunkId;
+  }
```

Comment:
Why have this setter? It's a single line and it's only called in one place. In line 359 we're also setting the variable directly